mapper-processor/src/main/java/com/datastax/oss/driver/internal/mapper/processor/entity/DefaultEntityFactory.java

```diff
@@ -111,31 +123,57 @@ public EntityDefinition getDefinition(TypeElement processedClass) {
         }
 
         String getMethodName = getMethod.getSimpleName().toString();
-        boolean regularGetterName = getMethodName.startsWith("get");
-        boolean booleanGetterName =
-            getMethodName.startsWith("is")
-                && (typeMirror.getKind() == TypeKind.BOOLEAN
-                    || context.getClassUtils().isSame(typeMirror, Boolean.class));
-        if (!regularGetterName && !booleanGetterName) {
+
+        // Skip a few false positives when we don't require a "get" prefix:
+        if (getterStyle == GetterStyle.SHORT
+            && !mutable
```

Comment:
why do we need `!mutable` check here?

Reply:
Indeed it's not strictly needed. But if the entity is mutable, the two methods would be skipped anyway because they don t have corresponding setters `setToString() / setHashCode()`, so the check is unnecessary.

Along the same lines, `getterStyle == GetterStyle.SHORT` is not mandatory either. I might just skip the two methods solely based on their names, that will be easier to follow.